tests/gemmini/gemmini-rocc-tests/mlps/gemmini_matmul_generator.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 13,
```

Comment:
Is this change at all relevant?

Reply:
I don't think so to me unless it was speeding up a test?